.github/workflows/ci.yml

```diff
@@ -41,6 +41,24 @@ jobs:
       - name: "Installs SoftHSM and execute tests"
         uses: ./.github/actions/ci_script
 
+  tests-kryoptic:
+    name: Run tests against Kryoptic
+    runs-on: ubuntu-latest
+    container: fedora:rawhide
+    steps:
+      - name: Install dependencies
+        run: dnf -y install git cargo clang-devel kryoptic
+      - uses: actions/checkout@v2
+      - name: Test script
+        env:
+          KRYOPTIC_CONF: /tmp/kryoptic.sql
+          PKCS11_SOFTHSM2_MODULE: /usr/lib64/pkcs11/libkryoptic_pkcs11.so
```

Comment:
I guess you could also define another name for the env variable (like `PKCS11_KRYOPTIC_MODULE`) and then check which one is set to know if the implementation is using softHSM or kyoptic in the tests

Reply:
Here I also did not want to break everything at once so I reused what was there.  I would probably prefer some name without specific project like TEST_PKCS11_MODULE if we should change it.

Reply:
Got it, we can change later :)